_includes/pillars.html

```diff
@@ -0,0 +1,32 @@
+<div class="row">
+    <div class="col-md-8 col-md-offset-2 text-center">
```

Comment:
Why are you using `col-md-8 col-md-offset-2` here? Shouldn't it be `col-md-12`?

Reply:
Never mind, I see that you're doing this instead of adding horizontal margin in the paragraph.